File: Week13-14/Java_Multithreaded_Processing/Person.java
/**
 * Person class representing a person with a name.
 * This is a simple data class used in the data processing system.
 */
public class Person {
    private String name;
    
    /**
     * Constructor to create a Person with a name.
     * @param name The name of the person
     */
    public Person(String name) {
        this.name = name;
    }
    
    /**
     * Gets the name of the person.
     * @return The person's name
     */
    public String getName() {
        return name;
    }
    
    /**
     * Returns a string representation of the person.
     * @return String representation
     */
    @Override
    public String toString() {
        return name;
    }
}

File: Week13-14/Go_Multithreaded_Processing/person.go
package main

// Person represents a person with a name.
// This is a simple data structure used in the data processing system.
type Person struct {
	Name string
}

// NewPerson creates a new Person with the given name.
func NewPerson(name string) *Person {
	return &Person{Name: name}
}

// String returns a string representation of the person.
func (p *Person) String() string {
	return p.Name
}

